src/actions/set-cookies.ts

```diff
@@ -26,3 +26,12 @@ export async function setConsentCookie(value: boolean) {
     sameSite: "strict",
   });
 }
+
+export async function clearAnalyticsCookies() {
+  const cookieStore = cookies();
+  cookieStore.getAll().forEach((cookie) => {
+    if (cookie.name === "_ga" || cookie.name.startsWith("_ga_")) {
+      cookieStore.delete(cookie.name);
+    }
+  });
```

Comment:
It feels wrong for us to be deleting cookies set by a third party but I don't think we have any other option! 😭

I'm worried we could be veering into a security issue here? Would some browsers flag us if we're "not staying in our lane"? 

It looks like `_ga`, `_gid`, `_gat` are additional cookie names to check for - could you add these in?

Maybe we should also add a link to a 'How to clear your cookies page' as a just in case we miss some? We might need to look into that a bit more though and make a ticket for it - we don't want to be writing/maintaining that!